consensus/types/src/beacon_block_body.rs

```diff
@@ -508,6 +518,60 @@ impl<E: EthSpec> From<BeaconBlockBody<E, FullPayload<E>>>
     }
 }
 
+impl<T: EthSpec> BeaconBlockBody<T> {
+    pub fn compute_merkle_proof(
```

Comment:
FYI - there's something similar (computing merkle proof) going on with blobs inclusion proof - it's not completely identical but there's probably some parts that are reusable - maybe worth renaming this function at least to avoid confusions 
https://github.com/sigp/lighthouse/pull/4900/files#diff-93ee4b8bd96daedfce7931df6ed17fcfe4dd8e9f75a8681fc0613caf71c55276R112-R180

Reply:
Renamed to `block_body_merkle_proof` (I think that name makes sense?).  

re: reusable code: 
In `kzg_commitment_merkle_proof` lines 158-180 can be replaced by `block_body_merkle_proof`. Just need to figure out how to make that function available within `BeaconBlockBodyRef<'a, T, Payload>` impl and `BeaconBlockBody` impl